index.js

```diff
@@ -152,10 +152,12 @@ module.exports = {
       // that version contains API to hook fastboot into ember-cli
 
       app.use((req, resp, next) => {
+        var fastbootQueryParam = (req.query.hasOwnProperty('fastboot') && req.query.fastboot === 'false') ? false : true;
+        var enableFastBootServe = !process.env.FASTBOOT_DISABLED && fastbootQueryParam;
```

Comment:
I think it would be nice if we'd start with `FASTBOOT_DISABLED=true` we'd still served can try fastboot for `http://localhost:4200/?fastboot=true` the usecase is non fastboot first development but we want to try stuff in fastboot too.